auth-service/src/main/java/dev/rckft/authservice/service/UserRegistrationService.java

```diff
@@ -25,7 +25,7 @@ public UserRegistrationService(UserRepository userRepository, PasswordEncoder pa
     public void register(UserRegisterRequest request) {
         String username = request.username();
         if (userExists(username)) {
-            LOGGER.debug("Unable to register user {}. User with given username already exists", username);
+            LOGGER.error("Unable to register user {}. User with given username already exists", username);
```

Comment:
Tutaj dałbym warning/info/debug, to jest przypadek biznesowy. Do error wrzucalbym przechwycone wyjatki/bledy systemowe